circle.yml

```diff
@@ -20,6 +20,11 @@ dependencies:
     - docker build -t oesteban/fmriprep .
 test:
   override:
+    # run unit tests
+    - source activate crnenv
```

Comment:
the workflow is containerized in a docker image. Thus, anything you want to do should be inside the image. To run code inside the image there are several strategies (see https://github.com/poldracklab/preprocessing-workflow/blob/master/build/circle_tests.sh#L6).

Basically, you may want to write an entry point for the docker image (like https://github.com/poldracklab/preprocessing-workflow/blob/master/build/files/run_fmriprep) in which you source the environment and call the tests. Finally, you will call the fmriprep docker image setting this new entrypoint (with something like ```--entrypoint=/usr/local/bin/run_shoshber_tests```). This will replace the default entrypoint ```run_fmriprep```).

Anything you place in that directory with the preffix ```run_``` will be placed and granted permissions in the docker image automatically.